packages/host/app/components/operator-mode/stack.gts

```diff
@@ -37,6 +37,16 @@ export default class OperatorModeStack extends Component<Signature> {
     for (let i = this.args.stackItems.length - 1; i > itemIndex; i--) {
       itemsToDismiss.push(this.args.stackItems[i]);
     }
+
+    // add dismissed animation on last item
+    const lastItemEl = document.querySelector(
+      `.operator-mode-stack > .inner > .item:last-child`,
```

Comment:
This is quite brittle. If dom structure changes in this path there will be no animation. Maybe you can add a special data attribute to the last item and target it here.

Reply:
I agree @jurgenwerk, by tackling this I added a new data attribute that contains the stack card index `data-stack-card-index` and targets the item with the last index